File: C++/프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    vector<int> answer;
    
    vector<int> query;
    
    int i, j, temp;
    
    
    for (int q=0; q<queries.size(); q++) {
        query = queries[q];
        i = query[0];
        j = query[1];
        
        temp = arr[j];
        arr[j] = arr[i];
        arr[i] = temp;
        
    }

    
    return arr;
}

File: JavaScript/프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.js
function solution(arr, queries) {
    var answer = [];
    
    for (const query of queries) {
        const i = query[0];
        const j = query[1];
        
        const temp = arr[j];
        arr[j] = arr[i];
        arr[i] = temp;
        
    }
    
    answer = arr;
    
    return answer;
}